ee/tabby-webserver/src/service/auth.rs

```diff
@@ -247,7 +247,9 @@ impl AuthenticationService for AuthenticationServiceImpl {
             password_hash(password).map_err(|_| PasswordResetError::Unknown)?;
 
         let user_id = self.db.verify_password_reset(code).await?;
```

Comment:
verify_password_reset should returns i64 as well?

Reply:
Already does? https://github.com/TabbyML/tabby/pull/1469/commits/d1aae1c8c444fac07698939ebf7b27ff2746c3f8

Reply:
then why it need another cast in 251?

Reply:
That's redundant, I suppose